app/views/OrderPage/index.tsx

```diff
@@ -0,0 +1,206 @@
+            <div className={styles.metaData}>
+                <Container
+                    className={styles.details}
+                    heading="Confirm Order"
+                >
+                    <div className={styles.headerDescription}>
+                        <div className={styles.quantity}>
+                            <TextOutput
+                                label="Book Name"
+                                value={title}
+                            />
+                        </div>
+                        <NumberInput
+                            name="quantity"
+                            label="Quantity"
+                            value={quantity}
+                            onChange={handleQuantityChange}
+                        />
+                        <TextOutput
+                            label="Price (NPR)"
+                            valueType="number"
+                            value={price}
+                        />
+                        <TextOutput
+                            label="Amount (NPR)"
+                            valueType="number"
+                            value={isDefined(quantity) ? price * quantity : 0}
+                        />
+                    </div>
+                </Container>
+                <div className={styles.orderButton}>
+                    <Button
+                        name={undefined}
+                        onClick={undefined}
+                        variant="secondary"
+                        icons={<IoCashOutline />}
+                    >
+                        Cash On Delivery
+                    </Button>
+                </div>
+            </div>
+        </div>
+    );
+}
+
+function OrderPage() {
+    const location = useLocation();
+    const params = new URLSearchParams(location.search);
+    const id = params.get('book');
+    const [quantity, setQuantity] = useState<number>(1);
+    const { data: result, loading } = useQuery<
+        BookByIdQuery,
+        BookByIdQueryVariables
+    >(BOOK_DETAIL, {
+        skip: !id,
+        variables: id ? { id } : undefined,
+    });
+
+    const alert = useAlert();
+
+    const [placeSingleOrder,
+        {
+            data: resp,
+            loading: submitting,
+        },
+    ] = useMutation<
+            SingleOrderMutation,
+            SingleOrderMutationVariables
+    >(
+        ORDER,
+        {
+            onCompleted: (response) => {
+                if (response?.placeSingleOrder?.ok) {
+                    alert.show(
+                        `Order #
+                        ${resp?.placeSingleOrder?.result?.orderCode}
+                        For Total Price: NPR
+                        ${resp?.placeSingleOrder?.result?.totalPrice}
+                        Order Status:
+                        ${resp?.placeSingleOrder?.result?.status}`,
+                        {
+                            variant: 'success',
+                        },
+                    );
```

Comment:
This may result in "undefined" text on alert.
We should avoid passing undefined values to formatted string